test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -93,6 +93,24 @@ def hook(context):
 
 @OnScenarioEnd
```

Comment:
that will create an screenshot AFTER the issue
so e.g. if the app crashes this will be the screen after the crash, meaning the app would not even be visible
Do we need (additionally?) the screenshot just before the problematic action? But that might be hard to achive

Reply:
If the app is not closed automatically after the crash then the screenshot is captured that shows the state of crash. For instance in https://github.com/owncloud/client/pull/9518#issuecomment-1072118187

Otherwise, the only option that I can think of right now is to capture the video of test run (anyway) and then delete it if there is no test error. I can not say that can be done easily.

Reply:
But this could be a good enhancement. I will open an issue. May be we can try it later.